content/articles/Factor-that-affects-the-performance-deep-learning-systems/index.md

```diff
@@ -0,0 +1,114 @@
+
+#### 3.  Technical expertise in deep learning system network design.
+The demand and need for deep learning experts is highly recommended here. Experience is needed to increase the learning rate and improve the system's performance.
+
+#### 4.  Which optimiser suits the deep network best?
+According to [research]( https://optimization.cbe.cornell.edu/index.php?title=Adam), the Adam optimiser offers the highest and best performance rate. The ability to outperform other approaches (optimisers) is lacking. 
+
+### How To Get A Perfect Performance Rate From A Deep Learning System
+>The performance rate of a deep learning system is focused on vision and pattern recognition.
+
+####  1. A Deep Neural Network System's Individual Units' (Layers') Functionality 
+
+In deep learning, [convolutional neural networks](https://reader.elsevier.com/reader/sd/pii/S1877050918308019?token=16B4CA6789714BCA2CEC6D44659321C7510FF2D8866A0E0CC6297D50E6129D2A4365D975C5F320F49E355B8662952541&originRegion=eu-west-1&originCreation=20211226214344) must take individual activation levels into account. This step, though, is difficult to achieve. First, the issue of [unit interpretability]( https://www.google.com/url?sa=t&source=web&rct=j&url=https://www.pnas.org/content/117/48/30071&ved=2ahUKEwiSvqrstIL1AhXVOuwKHTtTDZAQFnoECAYQAQ&usg=AOvVaw2Cnz9K-g6M2q__QYAUi3h1) in deep networks must be grasped.
+This knowledge will assist the network in avoiding becoming a completely opaque black box. Our measurement method is [network dissection](https://www.google.com/url?sa=t&source=web&rct=j&url=https://arxiv.org/pdf/1801.03454&ved=2ahUKEwiQua_b5oD1AhXPBWMBHQyaDHEQFnoECAoQAQ&usg=AOvVaw0oRGCGYRncsO9M6TZhGf3R) . This method calculates a system's deep learning performance rate.
+Measurements of alignments between different levels are also part of network dissection in the deep convolutional network (units). It is being done to separate dense data sets for a quicker performance rate. This segmentation notion is also known as "broden." Other classifier networks can use the Broden approach as well. The Progressive generative adversarial network [P-GAN](https://www.google.com/url?sa=t&source=web&rct=j&url=https://machinelearningmastery.com/introduction-to-progressive-growing-generative-adversarial-networks/&ved=2ahUKEwiK4dCV9YD1AhWK2hQKHblpDToQtwJ6BAgMEAE&usg=AOvVaw2AHbr7wD_no4DEy3LpvS7n) with the [VGG-16](https://www.google.com/url?sa=t&source=web&rct=j&url=https://towardsdatascience.com/step-by-step-vgg16-implementation-in-keras-for-beginners-a833c686ae6c%23:~:text%3DVGG16%2520is%2520a%2520convolution%2520neural,vision%2520model%2520architecture%2520till%2520date.%26text%3DIt%2520follows%2520this%2520arrangement%2520of,consistently%2520throughout%2520the%2520whole%2520architecture.&ved=2ahUKEwjn5ZWL84D1AhUDyoUKHXnMAJUQFnoECAYQBQ&usg=AOvVaw1PMy6ITxkEdXItU7nbohvO)  scene classifier. The most common are these two.
```

Comment:
How did we get to GANS all of a sudden from CNNs?

There is no flow between paragraphs.